### PR TITLE
Main footer

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

**Comment by sivakumar-kailasam** [comment]
@locks  I'm waiting on godfrey to get a dedicated bucket on s3 for this project but for now its on the `api-docs.emberjs.com` bucket. I've created a dedicated fastly service for this, so you don't need to worry about the s3 swap later by using the domain prefix https://glimmer-styleguide.global.ssl.fastly.net to refer to the assets. 


eg., https://glimmer-styleguide.global.ssl.fastly.net/glimmer-styleguide/enable-ci/index.html

**Comment by locks** [comment]
@sivakumar-kailasam We have a bucket for builds, dashboard, etc, where this should go to as well.

**Comment by sivakumar-kailasam** [comment]
Assuming its the `builds.emberjs.com` bucket that you're referring to? I've reconfigured the above service to create a generic fastly service, https://ember-builds.global.ssl.fastly.net/ for the builds bucket. But to get this repos' assets to be published there we might need help from someone who has keys with write access to that bucket😅 who can configure that on travis for us.

File: src/ui/components/main-footer/template.hbs [hunk 1]
```diff
@@ -16,18 +16,29 @@
...
-      <a href="https://github.com/emberjs/ember.js" title="GitHub">
-        <i class="icon-github"></i>
-      </a>
-      <a href="https://plus.google.com/communities/106387049790387471205" title="Google+">
-        <i class="icon-gplus"></i>
-      </a>
+    <div class="footer-contributions">
+      <div class="contributor">
+        <p>Hosted by:</p>
+        <a href="https://www.heroku.com/emberjs"><img src="/images/logos/vendors/heroku-logo.svg" class="logo-heroku"></a>
```
Comment (sivakumar-kailasam) [inline src/ui/components/main-footer/template.hbs]: shouldn't images be served along side these assets?

Comment (locks) [reply src/ui/components/main-footer/template.hbs]: they'll stay on the website for now

**Pull request merged**